src/Components/GameBoard.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from "react";
+import monkey from "../img/vector-monkey.png";
+import platypus from "../img/vector-platypus.png";
+import Board from "./Board/Board";
+import { Gameboard, generateBlankGameboard } from "./Board/Cell";
+import Menu from "./Menu/Menu";
+import Player from "./Player/Player";
+import PlayerField from "./Player/PlayerField";
+import Confetti from "./ScoreBoard/Confetti";
+import ScoreBoard from "./ScoreBoard/ScoreBoard";
+import { GameStatus } from "./Utils/GameStrategy";
+
+const P1 = new Player("Perry", platypus, "rgb(49, 224, 255)", "P1", 0);
+const P2 = new Player("Mino", monkey, "rgb(255, 164, 60)", "P2", 0);
+
+const blankGameboard = generateBlankGameboard();
+const GameBoard = function () {
+  const [gameboard, setGameboard] = useState(blankGameboard as Gameboard);
```

Comment:
add return type to `generateBlankGameboard` instead

Reply:
no need to create a const outside
```suggestion
  const [gameboard, setGameboard] = useState(generateBlankGameboard());
```

Reply:
fixed